pkg/security/probe/probe_windows.go

```diff
@@ -72,16 +76,28 @@ func (p *Probe) Start() error {
 					continue
 				}
 				ev.Type = uint32(model.ExecEventType)
+				if e != nil {
+					ev.Exec.Process = &e.Process
+				}
 			case stop := <-p.onStop:
 				log.Infof("Received stop %v", stop)
 				e = p.resolvers.ProcessResolver.GetProcessEntry(process.Pid(stop.Pid))
 				defer p.resolvers.ProcessResolver.DeleteProcessEntry(process.Pid(stop.Pid))
+
 				ev.Type = uint32(model.ExitEventType)
+				if e != nil {
+					ev.Exit.Process = &e.Process
+				}
 			}
 
 			if e != nil {
-
+				// use ProcessCacheEntry process context as process context
 				ev.ProcessCacheEntry = e
+				ev.ProcessContext = &e.ProcessContext
+				if ev.ProcessContext == nil {
+					panic("should always return a process context")
```

Comment:
Nit: I think a message like "Could not resolve process context of Start event" is more informative than the current message (**What** should always return?)